### PR TITLE
Fixes logging for services; simplifies zipkin-query; defers c* i/o

### PR DESCRIPTION
**Pull request by codefromthecrypt**
The log level flags for all processes were broken, defaulting to DEBUG
level. This resurrects them by using logback directly.

This also squashes some indirection in the query service in preparation
of removing ostrich completely.

Finally, this defers network i/o caused by c*'s Repository ctor.

### CODE REVIEW

File: zipkin-query-service/config/query-cassandra.scala [hunk 1]
```diff
@@ -37,18 +37,14 @@ if (username.isDefined && password.isDefined) {
...
-  handlers = List(ConsoleHandler())
-)
+val repository = new Repository(CassandraSpanStoreDefaults.KeyspaceName, cluster)
+val spanStore = new CassandraSpanStore(repository)
+val dependencies = new CassandraDependencyStore(repository)
 
 QueryServiceBuilder(
-  storeBuilder,
-  serverBuilder = ZipkinServerBuilder(serverPort, adminPort).loggers(List(loggerFactory))
+  "0.0.0.0:" + serverPort,
```
Comment (jamescway) [inline zipkin-query-service/config/query-cassandra.scala:41]: [non-blocking] is there any significance to 0.0.0.0 in this case? is the intention the same as localhost?

**Comment by jamescway** [comment]
:+1:  to :green_heart: build

File: zipkin-query-service/config/query-cassandra.scala (hunk 1 above)
Comment (codefromthecrypt) [inline zipkin-query-service/config/query-cassandra.scala:41]: This was making explicit what the previous value was. Neat, huh?

Comment (jamescway) [inline zipkin-query-service/config/query-cassandra.scala:41]: cool!

**Comment by codefromthecrypt** [comment]
red is because cassandra flaked again. the other green shows that this is mergable as these checks are redundant.

**Pull request merged**